<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tts_preferences</class>
 <widget class="QWidget" name="tts_preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <widget class="QCheckBox" name="use_bark">
     <property name="text">
      <string>Use Bark</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Language</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QComboBox" name="language_combobox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Gender</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QComboBox" name="gender_combobox">
        <item>
         <property name="text">
          <string>Male</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Female</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Voice</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="voice_combobox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="enable_tts">
     <property name="text">
      <string>Enable Text to Speech</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>language_combobox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>tts_preferences</receiver>
   <slot>language_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>voice_combobox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>tts_preferences</receiver>
   <slot>voice_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gender_combobox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>tts_preferences</receiver>
   <slot>gender_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_bark</sender>
   <signal>toggled(bool)</signal>
   <receiver>tts_preferences</receiver>
   <slot>use_bark_changed(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enable_tts</sender>
   <signal>toggled(bool)</signal>
   <receiver>tts_preferences</receiver>
   <slot>enable_tts_changed(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>language_changed(QString)</slot>
  <slot>gender_changed(QString)</slot>
  <slot>voice_changed(QString)</slot>
  <slot>use_bark_changed(bool)</slot>
  <slot>enable_tts_changed(bool)</slot>
 </slots>
</ui>
